site2/docs/security-tls-authentication.md

```diff
@@ -69,9 +70,9 @@ authenticationProviders=org.apache.pulsar.broker.authentication.AuthenticationPr
 
 ### ... on Proxies
 
-To configure proxies to authenticate clients, put the folling in `proxy.conf`, alongside [the configuration to enable tls transport](security-tls-transport.md#proxy-configuration):
+To configure proxies to authenticate clients, put the following in `proxy.conf`, alongside [the configuration to enable tls transport](security-tls-transport.md#proxy-configuration):
```

Comment:
What's the following content? If you know the info, specify it. If you're not sure about it, consult an eng.